### PR TITLE
TRUNK-5115-new-1c:Replace use of conditional logging with slf4j param…

### PR DESCRIPTION
**Pull request by gracebish**
Pull request title
TRUNK-5115-new-1c:Replace use of conditional logging with slf4j parameterized messages
JiraIssueNumber: JiraIssueTitle
TRUNK-5115:Replace use of conditional logging with slf4j parameterized message
Description of what I changed
Replace use of conditional logging with slf4j parameterized messages
 Issue I worked on
see https://issues.openmrs.org/browse/TRUNK-5115
## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/util/OpenmrsUtil.java [hunk 1]
```diff
@@ -64,6 +64,8 @@
 import javax.xml.transform.dom.DOMSource;
 import javax.xml.transform.stream.StreamResult;
 
+import liquibase.pro.packaged.T;
+import liquibase.pro.packaged.V;
```
Comment (HerbertYiga) [inline api/src/main/java/org/openmrs/util/OpenmrsUtil.java:68]: @gracebish  do we need this?

File: api/src/main/java/org/openmrs/util/OpenmrsUtil.java [hunk 2]
```diff
@@ -979,11 +979,9 @@ public static boolean deleteDirectory(File dir) throws IOException {
...
 			}
 			boolean success = f.delete();
-			
 			if (log.isDebugEnabled()) {
 				log.debug("   deleting " + f.getName() + " : " + (success ? "ok" : "failed"));
 			}
-			
 			if (!success) {
 				f.deleteOnExit();
 			}
```
Comment (gitcliff) [inline api/src/main/java/org/openmrs/util/OpenmrsUtil.java:987]: @gracebish thanks for the work
are there anychanges needed in this class ie` OpenmrsUtil` ,

Comment (gracebish) [reply api/src/main/java/org/openmrs/util/OpenmrsUtil.java:987]: @gitcliff  Any change on the file causes errors which also breaks mvn. Kindly advise on how to deal with this.

Comment (gitcliff) [reply api/src/main/java/org/openmrs/util/OpenmrsUtil.java:987]: you can manually fix them by adding back the deleted lines

File: api/src/main/java/org/openmrs/util/OpenmrsUtil.java [hunk 3]
```diff
@@ -64,8 +64,6 @@
 import javax.xml.transform.dom.DOMSource;
 import javax.xml.transform.stream.StreamResult;
 
-import liquibase.pro.packaged.T;
-import liquibase.pro.packaged.V;
```
Comment (ujjwaltyagi355) [inline api/src/main/java/org/openmrs/util/OpenmrsUtil.java]: Yaa, that was needed!! Great work @gracebish.

**Pull request merged**